pkg/bootstrapper/bootstrapper.go

```diff
@@ -179,7 +179,7 @@ func NewWinNodeBootstrapper(k8sInstallDir, ignitionFile, kubeletPath string, cni
 		}
 	}
 
-	// If there is already a kubelet service running, find and assign it
+	// If there is already a kubelet service running, find it
```

Comment:
Nit: this should be in the previous commit.

Reply:
done